modules/local/samplesheet_check.nf

```diff
@@ -0,0 +1,32 @@
+process SAMPLESHEET_CHECK {
```

Comment:
Can `check_sample_sheet.nf` be used? It's a little confusing to have 2 processes doing the same thing.